dataflow/gpu-examples/tensorflow-landsat-prime/e2e_test.py

```diff
@@ -60,8 +60,8 @@ def test_tensorflow_landsat(
     utils: Utils, bucket_name: str, run_dataflow_job: str
 ) -> None:
     # Wait until the job finishes.
-    timeout = 30 * 60  # 30 minutes
-    utils.dataflow_jobs_wait(job_name=utils.hyphen_name(NAME), timeout_sec=timeout)
+    job_id = utils.dataflow_job_id(utils.hyphen_name(NAME))
+    utils.dataflow_jobs_wait(job_id, timeout_sec=30 * 60)
```

Comment:
Consider changing 30 * 60 to a variable to avoid a magic number.

Reply:
Thanks, made into a constant at the top.